### PR TITLE
Dummy data

### PR DESCRIPTION
**Pull request by saltedseacorn**
What:

Create lib/game, teams, game_teams.
Create test/game_test, teams_test, game_teams_test.

Modify stat_tracker, managers, and individual classes to have vertical transversal.

Code was written live with all project members.

Please make notes or contact me for any further questions.

Great job all.

### CODE REVIEW

**Comment by gaelyn** [comment]
I think this is a great start y'all. Good idea Kon on going ahead and adding the teams and team_game classes.

**Comment by IcelandicGambit** [comment]
Everything makes sense to me. Relieved to start writing methods tomorrow.

**Comment by lcpulzone** [comment]
This looks great, thank you for driving tonight Kon.

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class StatTracker
 
   def self.from_csv(locations)
     StatTracker.new
-    @games = CSV.parse(File.read(locations[:games]), headers: true)
-    @game_teams = CSV.parse(File.read(locations[:game_teams]), headers: true)
-    @teams = CSV.parse(File.read(locations[:teams]), headers: true)
+    @games = CSV.parse( File.read( locations[:games] ), headers: true)
+    @game_teams = CSV.parse( File.read(locations[:game_teams] ), headers: true)
+    @teams = CSV.parse( File.read( locations[:teams] ), headers: true)
```
Comment (lcpulzone) [inline lib/stat_tracker.rb]: This looks great!  Glad this was figured out!

**Pull request merged**